frontend/packages/ceph-storage-plugin/src/components/dashboards/persistent-internal/status-card/whitelistedHealthChecks.ts

```diff
@@ -0,0 +1,5 @@
+// TODO(@rexagod): Add KCS/doc links once available.
+
+export enum whitelistedHealthChecksRef {
+  MON_DISK_LOW = '#',
+}
```

Comment:
```suggestion
export const whitelistedHealthChecksRef = {
  MON_DISK_LOW:  '#',
}
```
A key value object works better than enum.
Enums should be used for declaring constant data types for compile time checks

Reply:
Afreen suggested in an earlier [review](https://github.com/openshift/console/pull/9426#discussion_r673216428) to change it to enum. Should I change it back to an object?

Reply:
yes, object is best suited as it is alert name mapped to a link. Not a Data type.

Reply:
Yep, that was the originally what I went with as well. Updating.

Reply:
Updating.